### DISCUSSION TITLE
Docs tab breaking with MDX syntax

### DISCUSSION THREAD

**Discussion by zavjs** [mdx-js/mdx#1602] [Q&A] [answered]
Title (#icon) and comment are being left out from the rendered page. When I move to the docs page by clicking docs tab, it crashes. Any ideas why this is not working?

```jsx
import { ComponentProps } from "react"
import { Story, Canvas } from "@storybook/react"
import { ThemeProvider } from "src/ui/styled-components"
import { theme } from "src/ui/themes/theme"
import { Icon } from "./Icon"

<Meta title="MDX/Icon" component={Icon} />

export const icons = [
  "books",
  "settings",
  "notifications",
  "comment",
  "link",
  "arrow",
  "files",
  "home",
  "search",
  "arrowLeft",
  "plus",
  "sort",
  "filter",
  "arrows",
  "arrowRight",
  "checkCircle",
  "close",
  "user",
  "signout",
  "angleLeft",
  "angleRight",
]

# Icons

With `MDX`, we can define a story for `Checkbox` right in the middle of our Markdown documentation.

<Story name="all icons">
  {icons.map((icon) => (
    <div key={icon} title={icon}>
      <Icon name={icon} size="80px" color="#a7a7a7" />
    </div>
  ))}
</Story>
```

**Comment by ChristianMurphy** [comment, marked as answer]
MDX appears to parse this as expected https://astexplorer.net/#/gist/63b34a13a589b7192a6eede47459ae5b/098fac051a8fc4f809b1d1f8dc13db8f38ce440a :thinking: 

Could you share a runnable example maybe in CodeSandbox building off their Storybook template https://codesandbox.io/s/youthful-hugle-96sv7 ?
Have there been any discussions with the storybook maintainers which pointed to MDX in particular, over storybook or the module reloader?
